public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Also for @marden-erin: These pngs and the ico isn't needed; they're leftover from the template. I'd like you to take a look at how this is setup and do some research on your own and figure out how to put the logo and planet illustration you made into the project.

I'd recommend making another folder in src called `images` or something along those lines. Then put the images there and try to show them on the ExamplePage.

You got this!!